src/tests/webloader.test.js

```diff
@@ -0,0 +1,49 @@
+import * as assert from 'assert';
+import { WindowMock, DocumentMock } from './mocks/document.mock.js';
+import EvolvMock from './mocks/evolv.mock.js';
+
+let webloader;
+describe('the web loader', () => {
```

Comment:
I think this would more appropriately be an end-to-end test with something like TestCafe. You're really testing your mocks here more than you're testing if the web-loader works (if at all).